templates/apps/sumo-aws-apps/config/config.template.yaml

```diff
@@ -384,25 +489,9 @@ Resources:
             Resource: !Sub "arn:${AWS::Partition}:sns:${AWS::Region}:${AWS::AccountId}:*"
       Topics: [!Ref "SumoSNSTopic"]
 
-  WaitHandle:
-    Type: "AWS::CloudFormation::WaitConditionHandle"
-
-  EnableConfigWaitHandle:
-    Condition: enable_config
-    DependsOn: ConfigRole
-    Type: "AWS::CloudFormation::WaitConditionHandle"
-
-  EnableConfigWaitCondition:
-    Type: "AWS::CloudFormation::WaitCondition"
-    Properties:
-      Handle: !If [enable_config, !Ref EnableConfigWaitHandle, !Ref WaitHandle]
-      Timeout: "10"
-      Count: 0
-
-  DeliveryChannelWhenConfigIsTurnedOn:
```

Comment:
why did we delete config delivery channel is is not needed to export logs to S3 ?

Reply:
I replaced it with SumoCustomAWSConfig because the old code could not configure DeliveryChannel when AWSConfig was being used in another role.